### PR TITLE
Docs: Update grafana configuration documentation

### PR DESCRIPTION
**Pull request by rpriyanshu9**
**What is this feature?**

Update the grafana configuration documentation.

**Why do we need this feature?**

Enhance documentation clarity for better readability.

**Who is this feature for?**

An end user reading the documentation.

**Which issue(s) does this PR fix?**:

Fixes N/A (It's a documentation change)

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [x] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 1]
```diff
@@ -812,7 +812,7 @@ that this organization already exists. Default is 1.
 
 ### auto_assign_org_role
 
-The role new users will be assigned for the main organization (if the
+The role that new users will be assigned for the main organization (if the
 `auto_assign_org` setting is set to true). Defaults to `Viewer`, other valid
 options are `Admin`, `Editor` and `None`. e.g.:
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-grafana/_index.md:817]:
```suggestion
The `auto_assign_org` setting determines the default role assigned to new users in the main organization. The available options are `Viewer` (default), `Admin`, `Editor`, and `None`.
```

**Review by chri2547 — COMMENTED** [review]
Made a suggestion, please review to ensure I haven't changed the technical meaning of what you are trying to convey. Ping me for re-review, pls. Thank you very much for the contribution.

File: docs/sources/setup-grafana/configure-grafana/_index.md (hunk 1 above)
Comment (rpriyanshu9) [reply docs/sources/setup-grafana/configure-grafana/_index.md]: Hey @chri2547, thanks for the review. 

The configuration variable's name is `auto_assign_org_role` actually and it is only relevant when `auto_assign_org` is set to true. I've taken your suggestion into account and made the changes. Could you please take a look?

**Comment by rpriyanshu9** [comment]
@chri2547 I've made some changes, could you please take a look?

**Comment by rpriyanshu9** [comment]
pinging for a review, @chri2547!

**Review by chri2547 — APPROVED** [review]

**Pull request merged**